src/commands/LoadLogFile.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2020 FinancialForce.com, inc. All rights reserved.
+ */
+
+import { Context } from "../Context";
+import { Command } from "./Command";
+import { LogView } from "./LogView";
+import { appName } from "../Main";
+import { WebviewPanel } from "vscode";
+import { QuickPickWorkspace } from "../display/QuickPickWorkspace";
+import { GetLogFiles, GetLogFilesResult } from "../sfdx/logs/GetLogFiles";
+import { GetLogFile } from "../sfdx/logs/GetLogFile";
+import * as path from "path";
+import * as fs from "fs";
+import { Item, Options, QuickPick } from "../display/QuickPick";
+
+export class LoadLogFile {
+  static apply(context: Context): void {
+    new Command("loadLogFile", () => LoadLogFile.safeCommand(context)).register(
+      context
+    );
+    context.display.output(`Registered command '${appName}: Load Log'`);
+  }
+
+  private static async safeCommand(
+    context: Context
+  ): Promise<WebviewPanel | void> {
+    try {
+      return LoadLogFile.command(context);
+    } catch (err) {
+      context.display.showErrorMessage(`Error loading logfile: ${err.message}`);
+      return Promise.resolve();
+    }
+  }
+
+  private static async command(context: Context): Promise<WebviewPanel | void> {
+    const ws = await QuickPickWorkspace.pickOrReturn(context);
+    const logFiles = await GetLogFiles.apply(ws);
+    if (logFiles.status != 0)
+      throw new Error("Failed to load available log files");
+    const logFileId = await LoadLogFile.getLogFile(logFiles.result);
+    if (logFileId) {
+      const contents = await LoadLogFile.readLogFile(ws, logFileId);
+      return LogView.createView(ws, context, logFileId, contents[0], contents[1]);
+    }
+  }
+
+  private static async getLogFile(files: GetLogFilesResult[]): Promise<string | null> {
```

Comment:
The log are being ordered oldest top and newest at the bottom on the select a log view

Reply:
Have reversed this, thanks.